.github/workflows/ansible-test.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.9", "3.10", "3.11"]
-        ansible-version: [ stable-2.13, stable-2.14, stable-2.15]
+        ansible-version: [stable-2.13, stable-2.14, stable-2.15]
         exclude:
         # Python 3.11 is supported only from ansible-core 2.14 onwards
         - python-version: "3.11"
```

Comment:
- python-version: "3.11"
    ansible-version: stable-2.12

this above code need to remove